components/AddTask.js

```diff
@@ -1,25 +1,69 @@
-export default function AddTask() {
+import axios from "../utils/axios"
+import { API_URL } from "../utils/constants"
+import { useAuth } from "../context/auth"
+import React, { useState } from "react"
+import { Button, Container, Col, Row, Form } from "react-bootstrap"
+import { ToastContainer, toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+
+export default function AddTask(props) {
+  const { token } = useAuth()
+  const [tasks, setTasks] = useState([])
+  const [title, setTitle] = useState('')
+
+  function handleChange(event) {
+    const newTitle = event.target.value
+    setTitle(newTitle)
+  }
+
+  function notify(){
+    toast.success("Task added successfully!", {
+      position: "bottom-right",
+      autoClose: 3000
+    })
+  }
+
   const addTask = () => {
-    /**
-     * @todo Complete this function.
-     * @todo 1. Send the request to add the task to the backend server.
-     * @todo 2. Add the task in the dom.
-     */
+    axios({
```

Comment:
Should have checked for empty input

Reply:
I'm sorry i forgot to add this condition for add task function this time.